doc/reference/reference_lua/http.rst

```diff
@@ -1,353 +1,1141 @@
-                     > end
-                     > end
-
-            csrftoken cookie's Max-Age = 31449600
-            ---
-            ...
-
-            tarantool>
-
-        The following "shortcuts" exist for requests:
-
-        * ``http_client:get(url, options)`` -- shortcut for
-          ``http_client:request("GET", url, nil, opts)``
-        * ``http_client:post (url, body, options)`` -- shortcut for
-          ``http_client:request("POST", url, body, opts)``
-        * ``http_client:put(url, body, options)`` -- shortcut for
-          ``http_client:request("PUT", url, body, opts)``
-        * ``http_client:patch(url, body, options)`` -- shortcut for
-          ``http_client:request("PATCH", url, body, opts)``
-        * ``http_client:options(url, options)`` -- shortcut for
-          ``http_client:request("OPTIONS", url, nil, opts)``
-        * ``http_client:head(url, options)`` -- shortcut for
-          ``http_client:request("HEAD", url, nil, opts)``
-        * ``http_client:delete(url, options)`` -- shortcut for
-          ``http_client:request("DELETE", url, nil, opts)``
-        * ``http_client:trace(url, options)`` -- shortcut for
-          ``http_client:request("TRACE", url, nil, opts)``
-        * ``http_client:connect(url, options)`` -- shortcut for
-          ``http_client:request("CONNECT", url, nil, opts)``
-
-        Requests may be influenced by environment variables, for example
-        users can set up an http proxy by setting
-        :samp:`HTTP_PROXY={proxy}` before initiating any requests,
-        unless a ``proxy`` connection option has priority.
-        See the web page document
-        `Environment variables libcurl understands <https://curl.haxx.se/libcurl/c/libcurl-env.html>`_.
+    ..  _response_object-cookies:
 
-    .. _client_object-stat:
+    .. data:: cookies
 
-    .. function:: stat()
+        Response cookies.
+        The value is an array of two elements where the first one is the
+        cookie value and the second one is an array with the cookie's options.
+
+        :rtype: table
+
+        **See also:** :ref:`Cookies <response_cookies>`
+
+    ..  _response_object-body:
+
+    .. data:: body
+
+        A response body.
+        Use :ref:`decode <response_object-decode>` to decode the response body.
+
+        :rtype: table
+
+        **See also:** :ref:`Response body <response_body>`
+
+
+    ..  _response_object-proto:
+
+    .. data:: proto
+
+        An HTTP protocol version.
+
+        :rtype: string
+
+
+    ..  _response_object-decode:
+
+    .. function:: decode()
+
+        Decode the response body to a Lua object based on the content type.
+
+        :return: a decoded body
+        :rtype: table
+
+        **See also:** :ref:`Deserialization <response_deserialization>`
+
+        **Since:** :doc:`2.11.0 </release/2.11.0>`
+
+
+
+
+.. _io_object:
+
+io_object
+~~~~~~~~~
+
+..  class:: io_object
+
+    An IO object used to read/write data in chunks.
```

Comment:
Google [doesn't like](https://developers.google.com/style/slashes#slashes-with-alternatives) such use of the slash. Me neither %)